Product-page/product-page.css

```diff
@@ -0,0 +1,66 @@
+/* grid css */
+.grid-container {
+  display: grid;
+  grid-gap: 1rem;
+  height: 100vh;
+}
+
+.ecom-sidebar {
+  grid-column: 2 / 4;
+  grid-row: 2 / 6;
+  position:fixed;
+  width: 15rem;
+}
+
+.slider {
+  width: 100%;
+}
+
+
+
+.ecom-main {
+  grid-column: 2 / 6;
+  grid-row: 1 / 6;
+  display: grid;
+  height: 100vh;
+  grid-template-columns: repeat(auto-fit, minmax(250px, 1fr));
+}
+
+/* center using flex */
+
+.flex {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+/* center text */
+.center {
+  display: flex;
+  justify-content: center;
+}
```

Comment:
include both the classes in the utility class

Reply:
done